src/modules/launcher/PowerLauncher/Plugin/PluginManager.cs

```diff
@@ -53,7 +53,7 @@ public static List<PluginPair> AllPlugins
                         {
                             _allPlugins = PluginConfig.Parse(Directories)
                                 .Where(x => x.Language.ToUpperInvariant() == AllowedLanguage.CSharp)
-                                .GroupBy(x => x.ID) // Deduplicates plugins by ID, choosing for each ID the highest DLL product version. This fixes issues such as https://github.com/microsoft/PowerToys/issues/14701
+                                .GroupBy(x => x.ID) // DeDuplicates plugins by ID, choosing for each ID the highest DLL product version. This fixes issues such as https://github.com/microsoft/PowerToys/issues/14701
```

Comment:
Deduplicates should be an actual word. Was the spellchecker hitting it?

Reply:
Yes, see https://github.com/microsoft/PowerToys/pull/14868/files#diff-9b571d89da6e609011f053b6c5c9b374e3565ac198be344faf919fb991104fef. So should I add `deduplicates` to expect.txt?

Reply:
@jaimecbernardo?

Reply:
yes add it

Reply:
done